atqa-framework/.eslintrc.json

```diff
@@ -0,0 +1,70 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint",
+    "prettier",
+    "import",
+    "codeceptjs"
+  ],
+  "env": {
+    "node": true,
+    "codeceptjs/codeceptjs": true
+  },
+  "extends": [
+    "airbnb-typescript/base",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier",
+    "prettier/@typescript-eslint",
+    "plugin:codeceptjs/recommended"
```

Comment:
Why do we need to use airbnb typescript config?
I think that's enough:
```js
{
  "extends": [
    "eslint:recommended",
    "plugin:@typescript-eslint/eslint-recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:codeceptjs/recommended",
    "prettier"
  ]
}
```

Reply:
no idea what answer, see what is airbnb styleguide and why it cool

https://airbnb.io/javascript/

Reply:
We have our own code style and stick to the recommended rules configuration.
@faergeek right?